.github/workflows/ci.yml

```diff
@@ -181,7 +181,7 @@ jobs:
         run: |
           set -x
           cargo update -p indexmap --precise 1.6.2
-          cargo update -p hashbrown:0.12.2 --precise 0.9.1
+          cargo update -p hashbrown:0.12.3 --precise 0.9.1
```

Comment:
Uh, so does this need bumping every time hashbrown releases? 🤔

Reply:
Yes. In `rust-numpy`, I avoid this using Python scripting but that is not pretty either: https://github.com/PyO3/rust-numpy/blob/main/.github/workflows/ci.yml#L133-L143

Reply:
Doing it programatically makes sense; could potentially put that into the `noxfile` / `xtask` so we can just do `nox -s set-mrsv-versions` here?

Reply:
(We can leave this for a separate PR)